context/selection-of-signal-for-I.md

```diff
@@ -0,0 +1,165 @@
+
+# Selection of signal for I 
+
+**Status**: Draft
+
+_The purpose of this context document is not to argue for or against a particular topic, but rather to ensure that a proper discussion can be had by ensuring all participants in the discussion have the information they need to make informed opinions. It will also cover motivations, intentions, lore - things that have been discussed at length in the standards working group calls but never writen down. Do not treat this document as a specification, it's language is less formal, less accdemic, and the goal is shared understanding. Explain it like i'm 5 should be a good mantra here._
+
+## Abstract
+
+* The Software Carbon Intensity Specification is a way of scoring a software application for sustainability. [https://grnsft.org/sci](https://grnsft.org/sci)
+* Lower numbers are better than higher numbers, and reaching zero is impossible.
+* The equation is **SCI = (E * I + M) per R**
+* The **I** in the equation is the carbon intensity of electricity.
+* A discussion has arisen regarding the choice of emissions factor for **I** in the SCI equation. 
+* The choice of what to use for I acts as a signal.
+* The specification currently insists on marginal emission factors for **I**, and there is interest in understanding that original choice and exploring other emissions factors options.
+
+## Vocabulary
+
+* **LRMER**: Long-run marginal emission rates
+* **SRMER**: Short-run marginal emission rates
+* **AER**: Average emission rates

```

Comment:
As discussed with the working group and agreed with Henry, it would be helpful to move clarifying information to the beginning of the document.


``` suggestion

## Vocabulary and Context on Emission Factors

* There are different ways of calculating the carbon intensity of electricity per kWh of electricity. We are discussing three here:
    * ** AER (average emission rates)**: Total emissions divided by total electricity consumption, representing the carbon intensity of the entire electricity mix currently on the grid.
    * ** SRMER (short-run marginal emission rates)**: The emissions per unit change in electricity consumption, where the structure of the electrical grid (e.g., the generation, transmission, and distribution assets) is considered fixed. This represents the short-term change of system level emissions for a change in load.
    * **LRMER (long-run marginal emission rates)**: The emissions per unit change in electricity consumption, where the influence of the change in demand on both the operation and structure of the grid is taken into account. This represents the long-term change of system level emissions for a change of load.
* What average and short-run marginal indicate:
    * Average: The average quantifies the carbon intensity of electricity that is currently on the grid by taking into account all generators that produce electricity. The higher the share of green electricity on the grid, the lower the average carbon intensity and vice versa. The average does not provide information on which generator needs to turn on/off as a result of an action. 
    * Short-run Marginal: The marginal carbon intensity quantifies the carbon intensity of the power plant that increases its generation as a result of an action. The higher the marginal, the dirtier the next generator. It does not provide any information of power plants currently producing electricity and thus cannot inform users whether renewables are generating electricity or not. However, in rare cases of curtailment marginal emission factors are able to indicate renewable generation on the margin.   
* Generally speaking, whenever one refers to marginal emission rates, they mean short-run marginal emission rates. Only recently, the distinction between short run and long run marginal has been made.
* Both short-run marginal and average are available at an hourly, monthly and annual level in different regions or zones globally. 
* Both short-run marginal and average can be used for decision-making/load-shifting, but they each might signal a different time to consume electricity. This means that they can be used to shift load in time (e.g. different hour) and/or space (e.g. different location).
* So the choice of which signal to use has a significant impact on an SCI score as well as the impact of the action taken based on the SCI. Since the SCI is an action oriented spec the choice of one over the other can impact the actions that people take.
    * Short run marginal is the best signal to use if people want to make decisions to optimise the short term.  
    * Long run marginal is a better signal if you want actions people take to have an impact on a longer time horizon, for example to encourage more renewable plants to be built. It has far fewer benefits when viewed in the shorter term. According to NREL, it is impossible to calculate LRMER on an hourly basis in real-time and thus, proxies are needed.
    * [Average is considered to be a proxy for long run marginal](https://github.com/Green-Software-Foundation/sci/pull/353#issuecomment-1432307783), it’s easier to calculate and is correlated with long run marginal. Furthermore, AER strongly correlates with curtailment of renewables. Average emission factors perform poorly for estimating the short-term impact of interventions.
```

Reply:
I think curtailment is less 'rare' than indicated and an ample opportunity for emissions reductions.

Reply:
If we could reduce electricity sector emissions in California by nearly 5% by taking advantage of _already existing_ renewable generation, that would be a massive success. That is zero emissions power that is being wasted today. 5% is a massive amount of energy, last month alone California threw away nearly more than 600 GWh of clean energy.